enarx.ks

```diff
@@ -90,10 +90,88 @@ Type=ether
 [Network]
 DHCP=yes
 EOF
+
+get_maj_min() {
+    local _majmin
+    _majmin="$(stat -L -c '%t:%T' "$1" 2>/dev/null)"
+    printf "%s" "$((0x${_majmin%:*})):$((0x${_majmin#*:}))"
+}
+
+# give the /srv luks partition a partition label
+if ! [[ -b /dev/disk/by-partlabel/srv ]]; then
+   luksdev=/dev/$(ls -1 /sys/dev/block/$(get_maj_min /dev/disk/by-label/srv)/slaves)
+
+   # label the srv partition for later
+   sfdisk --part-label \
+       /dev/block/$(cat /sys/dev/block/$(get_maj_min $luksdev)/../dev) \
+       $(cat /sys/dev/block/$(get_maj_min "$luksdev")/partition) \
+       srv
```

Comment:
I don't understand why this branch is needed.

Reply:
I need `/dev/disk/by-partlabel/srv` to specify 
```
part /srv  --label=srv  --onpart=disk/by-partlabel/srv --encrypted --noformat --passphrase=temppass
```
on reinstalls... note the `--onpart=disk/by-partlabel/srv`